### PR TITLE
Add start option for bookings and events queries

### LINKED ISSUE #64: Return bookings which are after or on the current date

### PR DESCRIPTION
**Pull request by welvin21**
Closes #64

### CODE REVIEW

File: src/query/bookings.ts [hunk 1]
```diff
@@ -1,7 +1,18 @@
 import { Booking, FragmentableArray, Room } from "../generated/prisma-client";
+import moment from "moment";
 
-export const bookings = (parent, { data }, ctx): FragmentableArray<Booking> => {
-  return ctx.prisma.bookings({ where: data, orderBy: "start_ASC" });
+export const bookings = (
+  parent,
+  { data, start },
```
Comment (utkarsh867) [inline src/query/bookings.ts:6]: `start` is quite confusing here.

There is a `start` in `data` as well.

File: src/query/events.ts [hunk 2]
```diff
@@ -1,5 +1,16 @@
 import { Event, FragmentableArray } from "../generated/prisma-client";
+import moment from "moment";
 
-export const events = (parent, { data }, ctx): FragmentableArray<Event> => {
-  return ctx.prisma.events({ where: data, orderBy: "start_ASC" });
+export const events = (
+  parent,
+  { data, start },
```
Comment (utkarsh867) [inline src/query/events.ts:6]: There is a `start` field in `Event` as well. This makes the `start` argument quite confusing again.

**Comment by welvin21** [comment]
@utkarsh867 @98sean98  I'll change `owl` accordingly in a moment

**Comment by welvin21** [comment]
> @utkarsh867 @98sean98 I'll change `owl` accordingly in a moment

ahh, both `bookings` and `events` queries test failed on travis, because of the new `start_time` constraint. by default, it will return only `bookings` and `events` whose `start_time` is greater than or equal to the current time (`new Date()`).

**Comment by 98sean98** [comment]
Personally I find `start_after` or `after_start` or `start_limit` even clearer than `start_time`. I don't know what is best, what do you think in terms of api enduser?

**Comment by welvin21** [comment]
> Personally I find `start_after` or `after_start` or `start_limit` even clearer than `start_time`. I don't know what is best, what do you think in terms of api enduser?

well, `start_limit` sounds best to me personally, or something like `start_boundary`?

**Comment by utkarsh867** [comment]
@welvin21 Why is this pull request taking so long? Is there an issue with the code?

**Comment by welvin21** [comment]
> @welvin21 Why is this pull request taking so long? Is there an issue with the code?

@utkarsh867 i'm waiting for review and merge approval actually

**Comment by utkarsh867** [comment]
@98sean98 seems to be busy nowadays. So I'll go ahead and do it.

**Pull request merged**